packages/sdk-client/src/plugins/timezone/timezone.response.ts

```diff
@@ -21,6 +26,10 @@ const buggyFields: Record<string, string[]>  = {
   'UpdateSipTrunk': ['createTime', 'updateTime'],
   'GetSipTrunkById': ['createTime', 'updateTime'],
   'GetSipTrunks': ['createTime', 'updateTime'],
+  'CreateSipEndpoint': ['createTime'],
```

Comment:
😭 
One day this domain's dedicated "feature" adaptation should not be part of a generic plugin

Reply:
I've worked on it and it's removed starting PR #121